generatebundlefile/options.go

```diff
@@ -71,6 +72,7 @@ func NewOptions() *Options {
 	fs.StringVar(&o.outputFolder, "output", "output", "The path where to write the output bundle files")
 	fs.StringVar(&o.signature, "signature", "", "The Signature of the bundle to be added as an annotation")
 	fs.StringVar(&o.promote, "promote", "", "The Helm chart private ECR OCI uri to pull and promote")
+	fs.StringVar(&o.key, "key", "k", "The key to sign with")
```

Comment:
I did not read the docs about those parameters. Please validate that these are correct.